libafl/src/stages/calibrate.rs

```diff
@@ -104,8 +106,11 @@ where
         executor: &mut E,
         state: &mut E::State,
         mgr: &mut EM,
-        corpus_idx: CorpusId,
     ) -> Result<(), Error> {
+        let corpus_idx = state.current_corpus_idx()?.ok_or_else(|| {
```

Comment:
`let Some(corpus_idx) = ... else { return Err(...` could be more legible, I missed the questionmark and got confused for a second